Source/MediaInfo/Tag/File_Exif.cpp

```diff
@@ -1102,27 +1137,27 @@ static const char* Exif_IFD0_Compression_Name(int16u compression)
 {
     switch (compression)
     {
-    case 1: return "Uncompressed";
-    case 2: return "CCITT 1D";
-    case 3: return "T4/Group 3 Fax";
-    case 4: return "T6/Group 4 Fax";
+    case 1: return "Raw";
```

Comment:
As we are here, my understanding now is that it is common to and actually more about TIFF so let's remove the entire (duplicate) function and use:

```
char* Tiff_Compression(int32u Compression);
```

instead.